File: src/main/java/de/sean/splugin/spigot/inventories/FriendRemoveInventory.java
package de.sean.splugin.spigot.inventories;

import org.bukkit.Bukkit;
import org.bukkit.inventory.Inventory;

public class FriendRemoveInventory {
    public static final String INVENTORY_NAME = "Remove Friend";
    /// The content needs to be added when opened.
    public static Inventory inventory = Bukkit.createInventory(null, 9 * 3, INVENTORY_NAME);
}

File: SPlugin/common/src/main/kotlin/de/sean/splugin/bukkit/inventories/FriendRemoveInventory.kt
package de.sean.splugin.bukkit.inventories

import org.bukkit.Bukkit

object FriendRemoveInventory {
    const val INVENTORY_NAME = "Remove Friend"
    /// The content needs to be added when opened.
    var inventory = Bukkit.createInventory(null, 9 * 3, INVENTORY_NAME)
}
